vue/src/Tag/TagEdit.vue

```diff
@@ -534,6 +524,7 @@ export default defineComponent({
             this.onBlockTriggerChange();
 
             this.isDirty = false;
+            this.editTitle = `${translate('TagManager_EditTag')}${this.tag.typeMetadata?.name ? `: ${this.tag.typeMetadata.name}` : ''}`;
```

Comment:
This hides the colon if the type is empty, so:

`Edit tag: <type>`

Or:

`Edit tag`